bindings/ios/MEGASdk.mm

```diff
@@ -2160,6 +2160,34 @@ - (void)getPublicLinkInformationWithFolderLink:(NSString *)folderLink {
     self.megaApi->getPublicLinkInformation(folderLink.UTF8String);
 }
 
+#pragma mark - SMS
+
+- (SMSState)smsAllowedState {
+    return (SMSState)self.megaApi->smsAllowedState();
+}
+
+- (void)getRegisteredContacts:(NSArray<NSDictionary *> *)contacts delegate:(id<MEGARequestDelegate>)delegate {
+    MegaStringMap *stringMapContacts = MegaStringMap::createInstance();
+    for (NSDictionary *contact in contacts) {
+        NSString *key = contact.allKeys[0];
+        NSString *value = contact.allValues[0];
+        stringMapContacts->set(key.UTF8String, value.UTF8String);
+    }
+    self.megaApi->getRegisteredContacts(stringMapContacts, [self createDelegateMEGARequestListener:delegate singleListener:YES]);
```

Comment:
Another memory leak: the `MegaStringMap` is not deleted, and the `getRegisteredContacts()` doesn't take the ownership, but makes a copy. I'll fix it.

Reply:
Thanks Sergio for point it out. Hi @carolzato, please delete the temporary object `stringMapContacts` created here to fix the memory leak.

Reply:
Done